wordlespace.html

```diff
@@ -332,50 +37,45 @@
       let grays = getLetters($('#gray'), 'grayInput');
       let letterFreq = new Map();
       let words = wordlist
-          .map(w => w.toUpperCase())
-          // must contain all yellow letters somewhere
-          .filter(w => {
-            for (let i = 0; i < allYellows.length; i += 1) {
-              if (!w.includes(allYellows[i])) {
+        // must contain all yellow letters somewhere
+        .filter(w => {
+          let matchesYellows = allYellows.every(yellow => w.includes(yellow));
```

Comment:
why assign this to a variable, why not just in-line this in the `if(!...) return false`?

Reply:
I expected to use it later in the filter, but yeah no need